api/proto/info/info.proto

```diff
@@ -278,6 +278,20 @@ message ZCellularModuleInfo {
     string firmware_version = 3;
 }
 
+// Device Run State
+enum ZDeviceRunState {
+    // Just to handle Un-Initialized runState variable Internally
+    ZDEVICE_RUN_STATE_UNSPECIFIED = 0;
+    ZDEVICE_RUN_STATE_ONLINE = 1;
+    // Device Reboot in Progress. During bootup, Device reports "BOOTING"
+    ZDEVICE_RUN_STATE_REBOOTING = 2;
+    ZDEVICE_RUN_STATE_MAINTENANCE_MODE = 3;
+    // Device BaseOs Upgrade in Progress
+    ZDEVICE_RUN_STATE_BASEOS_UPGRADING = 4;
```

Comment:
s/UPGRADE/UPDATE/g - could be a downgrade.

We need to have more detailed semantics for this.
If I do zcli edge-node eveimage-update without setting activate, will the device report UPGRADING?
If I then activate, will it immediately report UPGRADING? (Note that the download and verification of the EVE image might take forever if the network is slow and it times out and retries).
Or does it report UPGRADING when it has committed to the update (and started shutting down the app instances)?
I think the last choice makes the most sense.

Reply:
Needs to be named UPDATING
Please search and replace upgrading to updating

Reply:
If I do zcli edge-node eveimage-update without setting activate, will the device report UPGRADING?
--> This workflow is unsupported and will go away anyways.

> Or does it report UPGRADING when it has committed to the update (and started shutting
> down the app instances)?

I think at the point where we no longer accept config changes and commit to the update is when we should report as UPGRADING. The BaseOs Status can be used to track the progress of the BaseOs Download / Verification anyways.. The Device State would be ONLINE and once Committed to Updating, ZDEVICE_STATE_BASEOS_UPDATING

Reply:
"Please search and replace upgrading to updating" --> This seems to be the only place in Info file that. used the term "UPGRADING".